### PR TITLE
Add MVCC performance recipe and live data column and field.

### PR DESCRIPTION
**Pull request by stbof**
DOC-4880
DOC-4987

### CODE REVIEW

**Review by maryliag — APPROVED** [review]
:lgtm:

> Reviewed 3 of 3 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15109#-:-NC64TT4-Z7WQ30qBXBw:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @kevin-v-ngo)

File: v22.2/performance-recipes.md [hunk 1]
```diff
@@ -167,6 +172,22 @@ ORDER BY total_reads ASC;
 
 Use the values in the `total_reads` and `last_read` columns to identify indexes that have low usage or are stale and could be dropped.
 
+### Too many MVCC versions
```
Comment (kevin-v-ngo) [inline v22.2/performance-recipes.md:175]: Should this be MVCC ( multi-**version** concurrency control) values (not "versions")?

Other places in our docs seem to refer to this as "MVCC values".

**Review by kevin-v-ngo — CHANGES_REQUESTED** [review]
Small change. Thanks for documenting recommendations as well!

**Review by kevin-v-ngo — APPROVED** [review]
Thanks!

File: v22.2/performance-recipes.md [hunk 2]
```diff
@@ -64,6 +64,11 @@ This section describes how to use CockroachDB commands and dashboards to identif
     <td><ul><li>Your tables have long write times.</li></ul></td>
     <td><ul><li><a href="#slow-writes">Remove unnecessary indexes.</a></li></ul></td>
   </tr>
+  <tr>
+    <td><ul><li>You experience high latency on queries that cannot be explained by high contention or a suboptimal query plan. You might also see high CPU on one or more nodes.</li></ul></td>
+    <td><ul><li>You are scanning over large numbers of <a href="architecture/storage-layer.html#mvcc">MVCC versions</a>. This is similar to how a full table scan can be slow.</li></ul></td>
```
Comment (taroface) [inline v22.2/performance-recipes.md:69]: I believe high latency + high CPU can be symptoms of other factors as well, such as workload concurrency exceeding CPUs. This is documented at https://www.cockroachlabs.com/docs/v22.1/common-issues-to-monitor.html#workload-concurrency - is it reasonable to cross-link to that section, and/or to mitigate the diagnosis with "You may be scanning over large numbers..."?

File: v22.2/performance-recipes.md [hunk 3]
```diff
@@ -167,6 +172,22 @@ ORDER BY total_reads ASC;
 
 Use the values in the `total_reads` and `last_read` columns to identify indexes that have low usage or are stale and could be dropped.
 
+### Too many MVCC values
+
+#### Indicators that your tables have too many MVCC values
+
+In the [Databases](ui-databases-page.html#tables-view) page in the DB Console, the Tables view show the percent of live data for each table. For example:
```
Comment (taroface) [inline v22.2/performance-recipes.md:179]:
```suggestion
In the [Databases](ui-databases-page.html#tables-view) page in the DB Console, the Tables view shows the percentage of live data for each table. For example:
```

File: v22.2/performance-recipes.md [hunk 4]
```diff
@@ -167,6 +172,22 @@ ORDER BY total_reads ASC;
...
+
+<img src="{{ 'images/v22.2/ui_databases_live_data.png' | relative_url }}" alt="Table live data" style="border:1px solid #eee;max-width:100%" />
+
+In this example, at `37.3%` the `vehicles` table would be considered to have a low percentage of live data. In the worst cases, the percentage can be `0%`.
+
+A low percentage can cause statements to scan more data ([MVCC values](architecture/storage-layer.html#mvcc)) than required, which can reduce performance.
```
Comment (taroface) [inline v22.2/performance-recipes.md:185]:
```suggestion
A low percentage of live data can cause statements to scan more data ([MVCC values](architecture/storage-layer.html#mvcc)) than required, which can reduce performance.
```

File: v22.2/ui-databases-page.md [hunk 5]
```diff
@@ -65,14 +68,15 @@ The **Overview** tab displays the SQL statements used to [create the table](crea
...
-- **Ranges**: the number of [ranges](architecture/glossary.html#architecture-range) in this table.
-- **Table Stats Last Updated**: the last time table statistics were created or updated.
-- **Auto Stats Collection**: whether [automatic statistics collection](cost-based-optimizer.html#table-statistics) is enabled.
-- **Regions/nodes**: the regions and nodes on which the table data is stored. This is not displayed on a single-node cluster.
-- **Database**: the database in which the table is found.
-- **Indexes**: the names of the indexes defined on the table.
+- **Size**: The approximate disk size of all replicas of this table on the cluster.
+- **Replicas**: The number of [replicas](architecture/replication-layer.html) of this table on the cluster.
+- **Ranges**: The number of [ranges](architecture/glossary.html#architecture-range) in this table.
+- **% of Live Data**: Percent of total uncompressed logical data that has not been modified (updated or deleted).
```
Comment (taroface) [inline v22.2/ui-databases-page.md:74]:
```suggestion
- **% of Live Data**: Percentage of total uncompressed logical data that has not been modified (updated or deleted).
```

**Review by taroface — CHANGES_REQUESTED** [review]
Some nits and a topical question/comment.

File: v22.2/performance-recipes.md (hunk 1 above)
Comment (stbof) [reply v22.2/performance-recipes.md]: Fixed.

**Review by taroface — APPROVED** [review]
LGTM!

**Pull request merged**